translations/de/hints/bryukh.html

```diff
@@ -1,40 +1,43 @@
 <body data-author="bryukh">
 <div class="question-answer">
-    <div class="question">
-        I have no idea how to start solving this mission
-    </div>
-    <div class="answer">
-        To iterate indexes in an array use "<a href="https://docs.python.org/2/library/functions.html#range">range</a>".
-<pre class="brush: python">
+  <div class="question">
+    Ich habe keine Ahnung wo ich mit der Lösung anfangen soll.
+  </div>
+  <div class="answer">
+    Um über Indizes eines Arrays zu iterieren, kannst Du die eingebaute Funktion
+    "<a href="https://docs.python.org/3.6/library/stdtypes.html#ranges">range</a>"
+    verwenden.
```

Comment:
das arme Original schlägt sich noch mit Version 2 herum. Ist das ein Überbleibsel oder Absicht?
Da gab es doch noch was mit xrange oder so. Kannst Du, wenn es Dir sinnvoll erscheint, eine Aktualisierung veranlassen/vorschlagen? Sorry I'm lazy and too far away anyway ;)